### PR TITLE
Filter by label

### PR DESCRIPTION
**Pull request by davidnortonjr**
Filter by container label(s) with wildcard support. Examples added via READMEs.

### CODE REVIEW

File: router/types.go [hunk 1]
```diff
@@ -118,6 +119,16 @@ func (r *Route) MatchContainer(id, name string) bool {
 	if err != nil || (r.FilterName != "" && !match) {
 		return false
 	}
+	for _,label := range r.FilterLabels {
+		labelParts := strings.Split(label, ":")
```
Comment (andrewgaul) [inline router/types.go]: Prefer `strings.SplitN(label, ":", 2)`

Comment (davidnortonjr) [inline router/types.go]: Updated, thanks @andrewgaul

**Comment by davidnortonjr** [comment]
Hi, is there anything you would like me to do in order to get this merged?

File: router/types.go [hunk 2]
```diff
@@ -118,6 +119,16 @@ func (r *Route) MatchContainer(id, name string) bool {
 	if err != nil || (r.FilterName != "" && !match) {
 		return false
 	}
+	for _,label := range r.FilterLabels {
+		labelParts := strings.SplitN(label, ":", 2)
+		labelKey := labelParts[0]
+		labelValue := labelParts[1]
```
Comment (mattaitchison) [inline router/types.go]: Need to check length of `labelParts` to avoid an `index out of range` panic

**Comment by davidnortonjr** [comment]
@mattaitchison, thanks for the input! Do you think this is ready to merge?

**Comment by irasnyd** [comment]
What is needed to get this merged? I really want this feature in my deployment.

**Comment by dblandin** [comment]
Hey! Just wondering if a release is planned that includes this feature? Thanks!

**Pull request merged**